curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-337-totient-stairstep-sequences.md

```diff
@@ -8,41 +8,41 @@ dashedName: problem-337-totient-stairstep-sequences
 
 # --description--
 
-Let {a1, a2,..., an} be an integer sequence of length n such that:
+Let $\\{a_1, a_2, \ldots, a_n\\}$ be an integer sequence of length $n$ such that:
 
-a1 = 6
+- $a_1 = 6$
+- for all $1 ≤ i &lt; n$ : $φ(a_i) &lt; φ(a_{i + 1}) &lt; a_i &lt; a_{i + 1}$
```

Comment:
```suggestion
- for all $1 ≤ i &lt; n$ : $φ(a_i) &lt; φ(a_{i + 1}) &lt; a_i &lt; {a_{i + 1}}^1$
```

https://projecteuler.net/problem=337

Reply:
I've moved part to which this `1` references (Euler's totient function) to the next line, so it shouldn't be needed anymore here.

Reply:
ahh - I see it now.